OPENSHIFT.md

```diff
@@ -0,0 +1,312 @@
+
+# Instructions
+
+## Install the Serverless Operator
+Install serverless operator from the repo https://github.com/openshift-knative/serverless-operator
```

Comment:
This will be updated once the autoscaler-keda component is available downstream.